main.py

```diff
@@ -1,51 +1,209 @@
-""" 
-A basic pygame template
 """
+-------------------------------------------------------------------------------
+Name:   malwaregame.py
+Purpose:  a game that generates a scenario that will allow for the user to figure out what the malware is and choose a solution that would best work for the malware that happens
  
-import pygame
+Author: Lue.Kyle
  
-# Define some colors
-BLACK    = (   0,   0,   0)
-WHITE    = ( 255, 255, 255)
-GREEN    = (   0, 255,   0)
-RED      = ( 255,   0,   0)
- 
-pygame.init()
-  
-# Set the width and height of the screen [width, height]
-size = (700, 500)
-screen = pygame.display.set_mode(size)
- 
-pygame.display.set_caption("My Game")
- 
-#Loop until the user clicks the close button.
-done = False
- 
-# Used to manage how fast the screen updates
-clock = pygame.time.Clock()
- 
-# -------- Main Program Loop -----------
-while not done:
-    # --- Main event loop
-    for event in pygame.event.get(): # User did something
-        if event.type == pygame.QUIT: # If user clicked close
-            done = True # Flag that we are done so we exit this loop
-  
-    # --- Game logic should go here
- 
-    # --- Drawing code should go here
-     
-    # First, clear the screen to white or whatever background colour. 
-    # Don't put other drawing commands above this, or they will be erased with this command.
-    screen.fill(WHITE)
-     
-    # --- Go ahead and update the screen with what we've drawn.
-    pygame.display.flip()
- 
-    # --- Limit to 60 frames per second
-    clock.tick(60)
-     
-# Close the window and quit.
-# If you forget this line, the program will 'hang'
-# on exit if running from IDLE.
-pygame.quit()
\ No newline at end of file
+Created: 26/03/2021
+------------------------------------------------------------------------------
+"""
+#Create title for the game
+print ("******Malware Detection Game******")
+print ("")
+#Add description
+print ("This bot will generate a scenario where a virus invades your computer. You wil have to determine the type of virus that has infected your computer and find out between two options on which one is better to help stop the virus.")
+print ("")
+
+#Generate adware scenario
+adware = input("One day, your computer starts to send different targeted ads. What type of malware is it?: ")
+if adware == "adware" or "Adware":
```

Comment:
You need to be explicit with your comparisons here (and all other similar areas in our code.  It should be:
```
if adware == "adware" or adware == "Adware":
```